docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -196,6 +204,32 @@ include::{sourcedir31}/clustered/client/src/test/resources/configs/docs/ehcache-
 Expiry in clustered caches work with an exception that `Expiry#getExpiryForAccess` is handled on a best effort basis for clustered tiers,
 it may not be as accurate as in the case of local tiers. Though the same is honored accurately in on-heap tiers.
 
+=== Clustered Unknown Inheritance
+
+We have included an option which allows a cache to be created without specifying its clustered resource pool allocation.
+In order to do so you must already have a cache created as either Shared or Dedicated.
```

Comment:
You wouldn't create the *cache* as Shared or Dedicated, right?   Not the cache but the pool for the cache?